packages/signals_core/lib/src/core/computed.dart

```diff
@@ -57,6 +57,8 @@ class _Computed<T> implements Computed<T>, _Listenable {
   bool _initialized = false;
   late T _value, _previousValue, _initialValue;
 
+  final SignalEquality<T>? equality;
```

Comment:
Maybe a global and local override?

Reply:
I think I am going to keep this as is since in most cases you want the default and every now and then a signal you will change the behavior.

Also I was going to make a global but it would lose the type safety